chanzuckerberg/single-cell-data-portal
# fix(wmg): Heat Map X/Y Scroll Bars and Getting Started Fix

## Reason for Change

- #4425

This PR moves the scrollbars to the edges of the screen and also fixes the "Step 1" box from covering gene labels if genes are selected first

## Changes

- add
- remove
- modify
    - Modify height and width values of heatmap content

## Testing steps

1. Select Tissue
2. Select Genes
3. Change window width until scrollbar appears
4. Ensure scrollbars are at edges of screen

Rdev: https://scroll-bars-frontend.rdev.single-cell.czi.technology/gene-expression

## Notes for Reviewer

Bulk of changes are in:
- frontend/src/views/WheresMyGene/components/GetStarted/style.ts
- frontend/src/views/WheresMyGene/components/HeatMap/style.ts

**Demo 1**

https://user-images.githubusercontent.com/109984998/228015729-82811bc8-522e-4669-8dc9-e9030afb25a9.mov
